src/components/chat/ask.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
it's fine here since it's a child of a client component, but i like to do 'use client' anyways since you're using state and hypothetically this component could be reused elsewhere